outputs.tf

```diff
@@ -26,14 +26,14 @@ output "alb" {
   value = aws_alb.alb
 }
 
-output "alb_target_group_green" {
-  value = aws_alb_target_group.green
-}
-
 output "alb_target_group_blue" {
   value = aws_alb_target_group.blue
 }
 
+output "alb_target_group_green" {
```

Comment:
(my OCD wants to put these outputs in the same order as shown in the outputs table 😅)